test/test_parabolic_2d.jl

```diff
@@ -94,19 +94,19 @@ isdir(outdir) && rm(outdir, recursive=true)
     )
   end
 
-  @trixi_testset "DGMulti: elixir_navier_stokes_convergence.jl" begin
-    @test_trixi_include(joinpath(examples_dir(), "dgmulti_2d", "elixir_navier_stokes_convergence.jl"),
+  @trixi_testset "DGMulti: elixir_navierstokes_convergence.jl" begin
+    @test_trixi_include(joinpath(examples_dir(), "dgmulti_2d", "elixir_navierstokes_convergence.jl"),
       cells_per_dimension = (4, 4), tspan=(0.0, 0.1),
-      l2 = [0.00153550768125133, 0.0033843168272696357, 0.0036531858107444067, 0.009948436427519428],
-      linf = [0.005522560467190019, 0.013425258500731063, 0.013962115643483375, 0.027483102120516634]
+      l2 = [0.0015355076812510957, 0.0033843168272696756, 0.0036531858107443434, 0.009948436427519214],
```

Comment:
Why did all the values change? Was the test case changed or is it just truncation error?

Reply:
It looks like it could be due to roundoff.

Reply:
It was all truncation error changes. I just updated the values to be safe.

Reply:
Thanks!